src/databricks/labs/ucx/progress/install.py

```diff
@@ -0,0 +1,51 @@
+import datetime as dt
```

Comment:
Note that this is part of "progress", but it will cover more than (migration) progress

Reply:
I am moving it